sdcm/nemesis.py

```diff
@@ -4347,8 +4347,8 @@ def disrupt_create_index(self):
                 raise UnsupportedNemesis(  # pylint: disable=raise-missing-from
                     "Tried to create already existing index. See log for details")
             try:
-                with adaptive_timeout(operation=Operations.CREATE_INDEX, node=self.target_node, timeout=7200) as timeout:
-                    wait_for_index_to_be_built(self.target_node, ks, index_name, timeout=timeout)
+                with adaptive_timeout(operation=Operations.CREATE_INDEX, node=self.target_node, timeout=7200):
```

Comment:
why duplicating the code for each nemesis and not adding only a single adaptive_timeout inside `wait_for_view_to_be_built` ?

Reply:
I don't think this is an issue. This way we have granularity for MV and INDEX operation - if we find they look the same and we decide to always use it, than it can go to `wait_for_view_to_be_built`